src/pg_to_evalscript/javascript_processes/sd.js

```diff
@@ -0,0 +1,50 @@
+function sd(arguments) {
+  const { data, ignore_nodata = true } = arguments;
+
+  if (data === undefined) {
+    throw new Error("Mandatory argument `data` is not defined.");
+  }
+
+  if (!Array.isArray(data)) {
```

Comment:
What error should be returned when the `data` is `null`?

`"Mandatory argument `data` is not defined."` or `"Argument `data` is not an array."`

In some of the other processes the check and the error are:
```javascript
if (data === null || data === undefined) {
    throw new Error(
      "Mandatory argument `data` is either null or not defined."
    );
  }
```